Makefile

```diff
@@ -10,17 +9,11 @@ py2:
 		exit 1; \
 	fi
 
-disco: py2
-	@if [[ -z '$(DISCO_HOME)' ]]; then \
-		echo -e "\n** Error: DISCO_HOME is not set. **\n"; \
-		echo -e "Please follow the installation instructions in README.txt\n"; \
-		exit 1; \
+celery: py2
+	@if [[ -z `pgrep celery` ]]; then \
```

Comment:
This needs to be `pgrep -f celery` on OS X, right now `celery` is started no matter what.